README.md

```diff
@@ -21,7 +21,12 @@ To increase Github API quota, acqure a Github token and store it:
 ```bash
 cat > secrets.yaml
 github_token: 'your-github-token'
-^D
+```
+
+If you would like to use reCAPTCHA, obtain a token and store it:
```

Comment:
Realised it won't work, as there is a matching key in `client/src/catalog-ondemand.html`